### ISSUE TITLE
Too many "Calling adzerk" messages

### ISSUE THREAD

**Issue by bqbn** [mozilla/addons-server#16015]
### Describe the problem and steps to reproduce it:

The olympia_web component generates tons of "Calling adzerk" messages, and I'm not sure how useful it is, but it is noise to other useful logs I usually look for.

```
Nov 17 02:01:41 ip-172-31-29-48 docker-addons[17073]: {"Timestamp": 1605578501515803136, "Type": "z.shelves", "Logger": "http_app_addons", "Hostname": "ip-172-31-29-48", "EnvVersion": "2.0", "Severity": 6, "Pid": 17, "Fields": {"uid": "<anon>", "remoteAddressChain": "106.78.83.251", "msg": "Calling adzerk"}, "severity": 200}
```

### What happened?
As described above.

### What did you expect to happen?
Please stop logging this message if possible.

### Anything else we should know?
n/a

**Comment by eviljeff** [comment]
@willdurand fixed this in #15994 ?

**Comment by willdurand** [comment]
yep
